crates/game/pong/src/game.rs

```diff
@@ -42,6 +42,18 @@ impl PongGame {
 }
 
 impl Game for PongGame {
+    fn from_model(model: core_crnn::thinking_layer::ThinkingLayer) -> Self {
+        PongGame::new(PongPlayer::model(model), PongPlayer::sync())
+    }
+
+    fn input_nodes() -> usize {
+        5
+    }
```

Comment:
Maybe male it 7:
1. enemy player y
2. enemy player velocity
3. ball x
4. ball y
5. ball dx
6. ball dy
7. own player y